tools.go

```diff
@@ -0,0 +1,8 @@
+//go:build tools
```

Comment:
I needed this on order to get the envtest tests running in github otherwise the installer does not find the tool